<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwidget</class>
 <widget class="QWidget" name="adminwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#0055ff;&quot;&gt;考试管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomelabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;欢迎你，管理员！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>学生数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="studentnumlabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>教师数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="teachernumlabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>700</width>
     <height>320</height>
    </rect>
   </property>
   <property name="title">
    <string>账户管理</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>300</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>新增账户</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="regnamelineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="regpswdlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="regrealnamelineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="studentradioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>学生</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="teacherradioButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>教师</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regpushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>300</width>
      <height>270</height>
     </rect>
    </property>
    <property name="title">
     <string>管理账户</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pswdlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="realnamelineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deletepushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>删除账户</string>
     </property>
    </widget>
    <widget class="QComboBox" name="useridcomboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changepushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>修改账户</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>195</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>身份：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="identifylineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>195</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
